### ISSUE TITLE
MoveCssInline removes XML namespaces

### ISSUE THREAD

**Issue by bernhof** [milkshakesoftware/PreMailer.Net#54]
Non-default XML namespaces are removed when inlining styles. For example:

    <html xmlns="http://www.w3.org/1999/xhtml" xmlns:v="urn:schemas-microsoft-com:vml" xmlns:o="urn:schemas-microsoft-com:office:office">

Turns into:

    <html xmlns="http://www.w3.org/1999/xhtml">

This is a problem since the specified namespaces support workarounds for image scaling in Outlook on Windows systems with >100% font scaling. See [this campaignmonitor](https://www.campaignmonitor.com/forums/post/30232/#p30229) thread for more info. Specifically, they enable the following conditional settings to apply in newer versions of Outlook:

``````
<head>
  <!--[if gte mso 9]><xml>
   <o:OfficeDocumentSettings>
    <o:AllowPNG/>
    <o:PixelsPerInch>96</o:PixelsPerInch>
   </o:OfficeDocumentSettings>
  </xml><![endif]-->
</head>
``````

XML namespaces were not removed in older versions of PreMailer.Net that used HtmlAgilityPack/Fizzler.

Specifying "html" in the `ignoreElements` argument of `MoveCssInline` unfortunately does not resolve the issue.

**Comment by niclasahlqvist** [comment]
Hello

Any plans on fixing this issue?

**Comment by martinnormark** [comment]
Hey @niclasahlqvist - can you confirm that you're on the latest version of PreMailer? I wonder if this issue was fixed with #102 - but perhaps it isn't.
